athina/loaders/function_loader.py

```diff
@@ -0,0 +1,64 @@
+from typing import List, Optional
+
+from athina.helpers.function_eval_util import get_named_non_default_parameters
+from athina.interfaces.athina import AthinaFilters
+from athina.interfaces.data import DataPoint
+from .loader import Loader
+from athina.evals.function.functions import operations
+
+
+class FunctionEvalDataPoint(DataPoint):
+    """Data point for a single Function invocation."""
+    keywords: Optional[List[str]]
```

Comment:
Why do we need this? 

Keywords will not be present in the datapoints, it will be defined in the `FunctionEvaluator` constructor args